skbio/io/format/embl.py

```diff
@@ -0,0 +1,1497 @@
+        new_data[new_key] = data[old_key]
+
+    # returning translated keys
+    return new_data
+
+
+# define a default textwrap.Wrapper for embl
+def _get_embl_wrapper(embl_key, indent=5, subsequent_indent=None, width=80):
+    """Returns a textwrap.TextWrapper for embl records (eg, write
+    <key>   <string> by providing embl key and a string. Wrap text to
+    80 column"""
+
+    # define the string to prepen (eg "OC   ")
+    prepend = '{key:<{indent}}'.format(key=embl_key, indent=indent)
+
+    # deal with 2° strings and more
+    if subsequent_indent is None:
+        subsequent_prepend = prepend
+
+    else:
+        subsequent_prepend = '{key:<{indent}}'.format(
+            key=embl_key, indent=subsequent_indent)
+
+    # define a text wrapper object
+    wrapper = textwrap.TextWrapper(
+        initial_indent=prepend,
+        subsequent_indent=subsequent_prepend,
+        width=width
+        )
+
+    return wrapper
+
+
+def _serialize_list(embl_wrapper, data):
+    """Serialize a list of obj using a textwrap.TextWrapper instance. Returns
+    one string of wrapped embl objects"""
+
+    # the output array
+    output = []
+
+    for line in data:
+        output += embl_wrapper.wrap(line)
+
+    # merge dates in one string
+    output = "\n".join(output) + "\n"
+
+    # return comupted string
+    return output
+
+
+# Method to determine if file is in EMBL format or not. A uniprot embl format
+# can't be parsed by this module (at the moment)
+@embl.sniffer()
+def _embl_sniffer(fh):
+    try:
+        line = next(_line_generator(fh, skip_blanks=True, strip=False))
+    except StopIteration:
+        return False, {}
+
+    try:
+        _parse_id([line])
+
+    except EMBLFormatError:
+        return False, {}
+
+    return True, {}
+
+
+@embl.reader(None)
+def _embl_to_generator(fh, constructor=None, **kwargs):
+    for record in _parse_embls(fh):
+        yield _construct(record, constructor, **kwargs)
+
+
+# Method to read EMBL data as skbio.sequence.DNA
+@embl.reader(Sequence)
+def _embl_to_sequence(fh, seq_num=1, **kwargs):
+    record = _get_nth_sequence(_parse_embls(fh), seq_num)
+    return _construct(record, Sequence, **kwargs)
+
+
+# Method to read EMBL data as skbio.sequence.DNA
+@embl.reader(DNA)
+def _embl_to_dna(fh, seq_num=1, **kwargs):
+    record = _get_nth_sequence(_parse_embls(fh), seq_num)
+    return _construct(record, DNA, **kwargs)
+
+
+# Method to read EMBL data as skbio.sequence.DNA
+@embl.reader(RNA)
+def _embl_to_rna(fh, seq_num=1, **kwargs):
+    record = _get_nth_sequence(_parse_embls(fh), seq_num)
+    return _construct(record, RNA, **kwargs)
+
+
+# No protein support at the moment
+@embl.reader(Protein)
+def _embl_to_protein(fh, seq_num=1, **kwargs):
+    # no protein support, at the moment
+    # record = _get_nth_sequence(_parse_embls(fh), seq_num)
+    # return _construct(record, Protein, **kwargs)
+    raise EMBLFormatError("There's no protein support for EMBL record")
```

Comment:
I think this is great to be explicit. When the issue is created (see comment above on the NOTE) would it be possible to add a mention of the issue into the exception message?